install.sh

```diff
@@ -59,6 +58,8 @@ execute_installation() {
     # and we are running inside the release clone
     # meaning the `--internal-install` flag is set.
 
+    legacy_transform_after_update || return 1
```

Comment:
Do we need to do these in dry-run mode? (same for before update above)

Reply:
Jup, totally correct! Thx

Reply:
`legacy_transform_before_update` will be executed always. This is I think critical. the others need a `if  if ! is_dry_run; then` around them